.github/workflows/release.yml

```diff
@@ -690,6 +690,8 @@ jobs:
   tag-nightly:
     if: ${{ inputs.auto }}
     needs: [push-tags, verify-docker-pull, verify-s3-download]
+    if: false
+    needs: [push-tags, verify-docker-pull, verify-s3-download, check-version]
```

Comment:
I have turned this off temporarily until I can e2e test the logic that will only trigger this for the current version